doc_src/disown.txt

```diff
@@ -0,0 +1,22 @@
+\section disown disown - remove a process from the list of jobs
+
+\subsection disown-synopsis Synopsis
+\fish{synopsis}
+disown [ PGID ... ]
+\endfish
+
+\subsection disown-description Description
+
+`disown` removes the specified <a href="index.html#syntax-job-control">job</a> from the list of jobs. Jobs are sent a hang-up signal when fish terminates, which usually causes the job to terminate; `disown` allows these processes to continue regardless.
+
+If no process is specified, the last job to be used is put in the foreground. If one or more `PGID` is specified, jobs with the specified process group IDs are removed from the job list. Invalid jobs are ignored and a warning is printed.
```

Comment:
It's the first job in the queue, which is the last constructed according to the comments in the `fg` code, so it's not necessarily the newest.